redi-connect-front/src/pages/app/me/About.tsx

```diff
@@ -1,27 +1,52 @@
 import React from 'react'
+import { Content } from 'react-bulma-components'
 import FormTextArea from '../../../components/atoms/FormTextArea'
+import FormInput from '../../../components/atoms/FormInput'
 import Editable from '../../../components/molecules/Editable'
 import { RedProfile } from '../../../types/RedProfile'
 import { connect } from 'react-redux'
 import { RootState } from '../../../redux/types'
 
+// import {
+//   menteeCountCapacityOptions
+// } from '../../../config/config'
+
 import {
   profileSaveStart
 } from '../../../redux/user/actions'
 import * as Yup from 'yup'
 
-import { FormikValues, FormikHelpers as FormikActions, useFormik, FormikProps } from 'formik'
+import { FormikValues, useFormik } from 'formik'
+
+// do we really need all these type???
+export type UserType =
+  | 'mentor'
+  | 'mentee'
+  | 'public-sign-up-mentor-pending-review'
+  | 'public-sign-up-mentee-pending-review'
+  | 'public-sign-up-mentor-rejected'
+  | 'public-sign-up-mentee-rejected';
 
 export interface AboutFormValues {
+  userType: UserType
   personalDescription: string
+  expectations: string
+  menteeCountCapacity: number
 }
 
 const validationSchema = Yup.object({
   personalDescription: Yup.string()
     .required()
     .min(100)
     .max(600)
-    .label('Personal description')
+    .label('Personal description'),
+  menteeCountCapacity: Yup.number().when('userType', {
+    is: 'mentor',
+    then: Yup.number()
+      .required('Please specify the number of mentees you can take on')
+      .min(1)
+      .max(3)
```

Comment:
Isn't this supposed to be two? In the current production Me page, mentors can only choose between 1 or 2.

Reply:
Ah, perhaps this is non-exclusive?

Reply:
Well according to the type it can be 1 or 2, according to form it was (is in prod ATM) 1 to 4. 
Make sense to define a single source of truth here. This is however not crucial

Reply:
Are we talking about the same form? I checked my "me" page on the production frontend, and I can only pick 1 or 2.

Some more background on this: Isabelle wants the ability to set more than 2, but only in the admin for "special cases" of mentors. We've done that for a few exceptional mentors that can take on many mentees. Otherwise, we want to limit the number of mentees a mentors can take on to keep things realistic. That's why we want mentors themselves to only be able to set 1 or 2.